apps/wallet/src/ui/app/pages/layout/index.tsx

```diff
@@ -27,7 +27,6 @@ function PageLayout({ forceFullscreen = false, children, className }: PageLayout
 			<div
 				className={cl('w-popup-width h-popup-height', st.container, className, {
 					[st.navHidden]: !isNavVisible,
-					'h-popup-height': true,
```

Comment:
Curious why we removed this

Reply:
it's already in `cl('....,h-popup-height'...` line 28